packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/identity_create/state/v0/mod.rs

```diff
@@ -32,6 +33,26 @@ use drive::grovedb::TransactionArg;
 use crate::execution::validation::state_transition::common::asset_lock::transaction::fetch_asset_lock_transaction_output_sync::fetch_asset_lock_transaction_output_sync;
 use crate::execution::validation::state_transition::common::validate_unique_identity_public_key_hashes_in_state::validate_unique_identity_public_key_hashes_in_state;
 
+#[repr(u64)]
+enum MinimalAssetLockValue {
+    V1 = 120000,
+}
+
+fn minimal_asset_lock_value(platform_version: &PlatformVersion) -> Result<u64, Error> {
+    match platform_version
+        .drive_abci
+        .validation_and_processing
+        .process_state_transition
+    {
+        0 => Ok(MinimalAssetLockValue::V1 as u64),
+        version => Err(Error::Execution(ExecutionError::UnknownVersionMismatch {
+            method: "minimal_asset_lock_value".to_string(),
+            known_versions: vec![0],
+            received: version,
+        })),
+    }
+}
```

Comment:
This should be in DPP, not Drive-ABCI. When thinking about where to put something, the question is this value a consensus value, if it is then it should be in DPP.

Reply:
Refactored